steal.lic

```diff
@@ -44,9 +44,21 @@ class Steal
     args = parse_args(arg_definitions)
 
     settings = get_settings
+
+    @target_province=''
+
+    if settings.hometown == 'Crossing'
```

Comment:
I wonder if we should change the stealing data to use hometown instead of province?

Alternatively, we could put this mapping in base-town.

What do you think?